fliplet-run.js

```diff
@@ -99,7 +99,7 @@ try {
 
 // Load packages for themes-only
 if (isTheme) {
-  sass = require('node-sass');
+  sass = require('sass');
```

Comment:
I assume you tested this locally with the theme component?

p.s. to have the `fliplet` command referencing your local CLI, run `sudo npm link` while in this repo. Npm will link your local copy to the `fliplet` executable.

Reply:
@squallstar I did run it. However, it had other issues not related to the switch away from `node-sass`.

![image](https://user-images.githubusercontent.com/290733/165726903-4d3b054e-30b5-480f-99e0-3046619c7eca.png)

That's because [variables like these](https://github.com/Fliplet/fliplet-theme-default/blob/master/scss/core/base.scss#L13) take value from theme configurations, which `fliplet-run` isn't providing.

It looks like the CLI tool for theme development and testing hasn't been useful for a while and we'll need to enhance it.

However, the fact that it was able to generate this error was good enough for me to consider it a move away from `node-sass`.

Reply:
@tonytlwu where are those variables coming from?

Reply:
@squallstar In real compilations, they come from either the default values set in theme.json ([link](https://github.com/Fliplet/fliplet-theme-default/blob/master/theme.json)), or settings found in the theme instance. I believe our compilation engine use these information to assign `$key: value` and get them compiled.

What we'll have to do to get the CLI tool useful for theme compilation again is to get the variables and default values from theme.json so that these variables do have values for the CLI tool.